packages/react-utils/src/components/Resource404/index.tsx

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
 import { Result } from 'antd';
 import { ExtraLinks, UtilPageExtraProps, extraLinksDefault } from '../UtilPageExtra';
-import lang from '../../lang';
+import { useTranslation } from '../../mls';
 
 /** typings for the resource404 component */
 interface Resource404Props extends UtilPageExtraProps {
```

Comment:
I feel like this should not take optional props. This is to avoid calling it mistakenly. The type checking should happen where it's being called

Reply:
made it optional for backwards compatibility, making them required would further require me to update everywhere they are used.